graphicsmagick/graphicsmagick.go

```diff
@@ -29,23 +30,41 @@ const (
 //
 // Params:
 //
-// - width / height: sizes for "-resize" argument (both optionals)
```

Comment:
Why did you move this line?
I think `width` and `height` are the most important parameters.